admin_app/src/components/ContentModals.tsx

```diff
@@ -111,34 +118,67 @@ export const EditModal: React.FC<EditModalProps> = ({
             </h3>
             <XMarkIcon className="w-4 h-4 float-right" onClick={onClose} />
           </div>
-          <div className="relative p-2 flex flex-auto">
+          <div className="relative px-5 py-1 flex flex-auto mb-2">
             <input
               id="content_title"
               type="text"
-              className="shadow appearance-none border active:outline-none border-neutral-400 text-sm rounded w-full dark:bg-gray-800 py-4 px-4 text-gray-800 dark:text-gray-400 overflow-auto required"
-              defaultValue={cardToEdit?.content_title}
+              className="shadow appearance-none border active:outline-none border-neutral-400 text-sm rounded w-full dark:bg-gray-800 py-3 px-3 text-gray-800 dark:text-gray-400 overflow-auto required"
+              defaultValue={cardToEdit?.content_title || ""}
```

Comment:
This is because `cardToEdit` is of type `ContentDataInEdit` i.e. its properties can be null (in cases where we're creating new data).

We can't pass null values to `defaultValue`, so I define what to show (`""`) in case it's null.
@sidravi1